application/Core/MORR/Session/SessionManager.cs

```diff
@@ -136,5 +137,15 @@ public void Process(IEnumerable<DirectoryPath> recordings)
 
             moduleManager.NotifyModulesOnSessionStop();
         }
+
+        private void LoadUsingConfig(FilePath configurationPath)
+        {
+            configurationManager.LoadConfiguration(configurationPath);
+
+            encoders = Encoders.Where(x => Configuration.Encoders.Contains(x.GetType()));
+            decoders = Decoders.Where(x => Configuration.Decoders?.Contains(x.GetType()) ?? false);
```

Comment:
Instead of doing this every time we start a recording or start processing, could we do this only once and then just call `IModuleManager.InitializeModules` in `ISessionManager.StartRecording` and `ISessionManager.Process`? I don't think we have a requirement to allow the configuration to be changed during runtime, so parsing it multiple times feels redundant. This would also mean we wouldn't have to keep passing the configuration path in the respective methods.

Reply:
Should be resolved now